working/augmentation-libraries/feature-specification.md

```diff
@@ -719,7 +767,24 @@ class C {
 }
 ```
 
-### Augmenting enum members
+For purposes of [signature matching][], the implicit setter induced by a
+non-final abstract variable has a positional parameter named `_`. *This means
```

Comment:
Also for non-abstract variables.
```dart
int x = 42;

@important
augment set x(value);
```
This should be valid. Otherwise we need to say what the name is, so that someone might be able to match it, or we force all augmentations to use `_`.

Reply:
It's not allowed. When I simplified the proposal, I said:

> Variable declarations themselves can't be augmented (by variables, getters, or setters, even if the getter or setter is incomplete), with the exception of abstract instance variable declarations.

We could loosen this and allowing augmenting variable declarations with incomplete getters and setters, but I don't think it's particularly useful and probably opens the door to more annoying edge cases.